baremaps-geoparquet/src/main/java/org/apache/baremaps/geoparquet/GeoParquetReader.java

```diff
@@ -218,16 +226,16 @@ private ParquetReader<GeoParquetGroup> createParquetReader(FileStatus file)
 
     @Override
     public Spliterator<GeoParquetGroup> trySplit() {
-      // Create a new spliterator by polling the next file
-      FileStatus file = queue.poll();
+      // Create a new spliterator by polling the next polledFileStatus
```

Comment:
I think the refactoring of the variable name affected this comment.

Reply:
yes but it was intended (it is the name of the variable in line 230) as queue.poll returns a FileStatus that was polled from the queue, hence polledFileStatus. I wasn't going to call it FileStatus nor file (they were instance variables)